Datasnakes/Orthologs/Blast/comparative_genetics_objects.py

```diff
@@ -21,21 +21,42 @@
 
 
 class CompGenObjects(object):
+    """ Comparative Genetics Analysis.
```

Comment:
Remove these docstrings.  They are copies of the _init_ docstings

Reply:
@sdhutchins Idk if I can do this myself or not on your PR?  Or if it's better if you do it?